<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1450</width>
    <height>890</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>600</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1300</width>
    <height>840</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_tittle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>241</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>极端天气去除雨雪系统</string>
    </property>
   </widget>
   <widget class="QComboBox" name="box_cap">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>95</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string> 0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> 4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> 5</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_cap">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>摄像头编号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_w">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>图像宽度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_h">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>图像高度</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_wxh">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>预设分辨率</string>
    </property>
   </widget>
   <widget class="QComboBox" name="box_wxh">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>125</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>1920x1080</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1600x900</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1280x720</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>960x540</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1920x1440</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1600x1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1280x960</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>800x600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>640x480</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1440x900</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1280x800</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="line_w">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>155</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1920</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_h">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>185</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1080</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtn_line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>丝状雨</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtn_point">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>滴状雨</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtn_snow">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>大雪</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_setclass">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>极端天气类别</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_setimg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>视频设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>760</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>保存并应用</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="text_info">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>760</y>
      <width>1141</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_info">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>730</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>输出信息</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_wxh">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QCheckBox" name="check_wh">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>165</y>
      <width>16</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>800</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_window">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>1140</width>
      <height>660</height>
     </rect>
    </property>
    <property name="text">
     <string>label_window</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_captest">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>95</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>检测</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_window2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>181</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>label_window2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_window3">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>40</y>
      <width>181</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>label_window3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>800</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_imagederain">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>800</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>图像去雨</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_precision">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>图像质量</string>
    </property>
   </widget>
   <widget class="QComboBox" name="box_precision">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>215</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>标清</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>高清</string>
     </property>
    </item>
   </widget>
   <zorder>btn_imagederain</zorder>
   <zorder>label_tittle</zorder>
   <zorder>box_cap</zorder>
   <zorder>label_cap</zorder>
   <zorder>label_w</zorder>
   <zorder>label_h</zorder>
   <zorder>label_wxh</zorder>
   <zorder>box_wxh</zorder>
   <zorder>line_w</zorder>
   <zorder>line_h</zorder>
   <zorder>rbtn_line</zorder>
   <zorder>rbtn_point</zorder>
   <zorder>rbtn_snow</zorder>
   <zorder>label_setclass</zorder>
   <zorder>label_setimg</zorder>
   <zorder>btn_save</zorder>
   <zorder>text_info</zorder>
   <zorder>label_info</zorder>
   <zorder>check_wxh</zorder>
   <zorder>check_wh</zorder>
   <zorder>btn_start</zorder>
   <zorder>label_window</zorder>
   <zorder>btn_captest</zorder>
   <zorder>label_window2</zorder>
   <zorder>label_window3</zorder>
   <zorder>btn_close</zorder>
   <zorder>label_precision</zorder>
   <zorder>box_precision</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1450</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>模型设置</string>
    </property>
    <addaction name="action_line"/>
    <addaction name="action_point"/>
    <addaction name="action_snow"/>
    <addaction name="separator"/>
    <addaction name="action_reset"/>
    <addaction name="separator"/>
    <addaction name="action_showfile"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>读取文件</string>
    </property>
    <addaction name="action_video"/>
    <addaction name="action_image"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="action_one"/>
    <addaction name="action_double"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_lite">
   <property name="text">
    <string>Lite Rain</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="action_line">
   <property name="text">
    <string>雨丝模型</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="action_point">
   <property name="text">
    <string>雨滴模型</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="action_reset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="action_video">
   <property name="text">
    <string>Video</string>
   </property>
  </action>
  <action name="action_image">
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="action_one">
   <property name="text">
    <string>One</string>
   </property>
  </action>
  <action name="action_double">
   <property name="text">
    <string>Double</string>
   </property>
  </action>
  <action name="action_origin">
   <property name="text">
    <string>Origin</string>
   </property>
  </action>
  <action name="action_showfile">
   <property name="text">
    <string>Show File</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="action_snow">
   <property name="text">
    <string>去雪模型</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
